new_project/examples/separate_apps/src/taker.ts

```diff
@@ -0,0 +1,96 @@
+import { SwapRequest } from "comit-sdk";
+import { formatEther } from "ethers/utils";
+import moment from "moment";
+import readLineSync from "readline-sync";
+import { toBitcoin, toSatoshi } from "satoshi-bitcoin-ts";
+import { Actor, checkEnvFile, startClient } from "./lib";
+
+(async function main() {
+    checkEnvFile(process.env.DOTENV_CONFIG_PATH!);
+
+    const taker = await startClient(1);
+
+    console.log(
+        "[Taker] Bitcoin balance: %f. Ether balance: %f",
+        parseFloat(await taker.bitcoinWallet.getBalance()).toFixed(2),
+        parseFloat(
+            formatEther(await taker.ethereumWallet.getBalance())
+        ).toFixed(2)
+    );
+
+    const peerId = readLineSync.question("What is the Maker's Peer Id?");
+    const addressHint = readLineSync.question(
+        "What is the Maker's Address Hint?"
+    );
+
+    const swapMessage = createSwap(taker, peerId, addressHint);
+
+    const takerSwapHandle = await taker.comitClient.sendSwap(swapMessage);
+
+    const actionConfig = { timeout: 100000, tryInterval: 1000 };
+
+    console.log(
+        "Swap started! Swapping %d Ether for %d %s",
+        formatEther(swapMessage.alpha_asset.quantity),
+        toBitcoin(swapMessage.beta_asset.quantity),
+        swapMessage.beta_asset.name
+    );
+
+    readLineSync.question("1. Continue?");
+
+    console.log(
+        "Ethereum contract funded! TXID: ",
+        await takerSwapHandle.fund(actionConfig)
+    );
+
+    readLineSync.question("3. Continue?");
+
+    console.log(
+        "Bitcoin redeemed! TXID: ",
+        await takerSwapHandle.redeem(actionConfig)
+    );
+
+    console.log("Swapped!");
+    console.log(
```

Comment:
:upside_down_face: For symmetry with `maker.ts`.
```suggestion
    
    console.log(
```